src/aspire/abinitio/commonline_sync3n.py

```diff
@@ -848,7 +872,7 @@ def _J_sync_power_method(self, Rijs):
 
         # Initialize candidate eigenvectors
         n_Rijs = Rijs.shape[0]
-        vec = randn(n_Rijs, seed=self.seed)
+        vec = rand(n_Rijs, seed=self.seed)
```

Comment:
Just curious, did this make a difference?

Reply:
For the small unit test case either return the same signs at the end.

However, they do generate relatively different vectors `vec` which _could_ change signs for other problems or epsilon criteria.

I'd rather just keep it as the original code had it unless we have a meaningful reason to change it. In other words, since we can't test every possibility, let's go with what is most likely to yield similar results...

Reply:
Ah ok. Yeah, sounds good. Thanks.